### ISSUE TITLE
I'm getting a "Failed to connect to api.github.com" Error after integration starts

### ISSUE THREAD

**Issue by rolandoam** [modcloth-labs/XBotBuilder#14]
Hi, I'm getting this error after trying to initiate integration on a newly created bot:

```
Bot Issue: error. Build Service Error.
Issue: Failed to connect to api.github.com: Connection refused (-1).
Integration Number: 1.
Integration URL: https://REDACTED/xcode/bots/C373308/integrations
Description: Failed to connect to api.github.com: Connection refused (-1).
```

Tracing the code in Xcode, the problem is that the xbot returned is nil and the error status is "checkout-error" (printed [here](https://github.com/modcloth-labs/XBotBuilder/blob/master/XBotBuilder/XBotBuilder/GitHubCommitStatus.swift#L31))

Not sure why there's a checkout error or what's the flow, I can manually checkout the code on my computer and I can manually create a bot for this project, it's just not working with XBotBuilder...

**Comment by gustavosaume** [comment]
Have you found any solution? I'm stuck at the same place. The result of every integration is connection refused.

**Comment by rolandoam** [comment]
yes, it was a bug in the app: it was trying to clone from `git@api.github.com`, I had to manually change the code in `GitHubXBotSync.swift:191`, to something like this:

``` swift
                gitUrl:"git@github.com:\(gitHubRepo.repoName).git",
```

**Comment by rolandoam** [comment]
btw, this seems to be fixed by PR #13

**Comment by gustavosaume** [comment]
Awesome, Thanks! I was looking  at the network traffic with a proxy but I must've missed that. Hopefully this gets pulled into master soon.
